app/src/main/res/values/strings.xml

```diff
@@ -409,7 +409,7 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_cycleway_value_lane_soft">cycle lane (dashed markings)</string>
     <string name="quest_cycleway_value_none">none</string>
     <string name="quest_cycleway_value_shared">lane shared explicitly with other traffic</string>
-    <string name="quest_buildingLevels_answer_multipleLevels">differs per building part</string>
+    <string name="quest_buildingLevels_answer_multipleLevels">Differs per building part</string>
```

Comment:
well this change is okay AFAIK (you cannot change the origin strings in poeditor)

But as for all translations, see https://github.com/westnordost/StreetComplete/blob/master/CONTRIBUTING.md#translating-the-app